polynote-kernel/src/main/scala/polynote/kernel/logging/package.scala

```diff
@@ -18,6 +18,16 @@ package object logging {
       blocking: Blocking.Service => new Logging.Service.Default(System.err, blocking)
     }
 
+    object Verbosity extends Enumeration {
```

Comment:
Not a big fan of `Enumeration` but I guess it gives you the ordering for free in this case.